build/app_template_files/__NativeSwiftTemplateAppName__/Podfile

```diff
@@ -2,10 +2,21 @@
 # platform :ios, '8.0'
 
 target '__NativeSwiftTemplateAppName__' do
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/wmathurin/SalesforceMobileSDK-iOS-Specs.git'
```

Comment:
That's why you need the CocoaPods specs repo as well (first source).
When looking for a pod, it tries the sources in order (kind of like a classpath).